xapian-maintainer-tools/profiling/profiler.c

```diff
@@ -65,7 +65,7 @@ int open(const char *pathname, int flags, ...)
 
 // wrapper for close()
 
-typedef int (*real_close_t)(int);
+typedef int(*real_close_t)(int);
```

Comment:
Hmm, are these changes in response to complaints from `xapian-check-patch`?

If so, I think it's `xapian-check-patch` that is buggy here and your original is better.

Reply:
Yes, these were being rejected by `xapian-check-patch`.
I updated the PR, now it will probably be failing both the checks.

Reply:
> Yes, these were being rejected by `xapian-check-patch`.

OK, I'll fix that.

> I updated the PR, now it will probably be failing both the checks.

Yes, though it's also failing due to an indentation issue.